src/main/java/org/parabot/Landing.java

```diff
@@ -29,6 +29,7 @@ public static void main(String... args) throws IOException {
 
         parseArgs(args);
 
+
```

Comment:
Unnecessary empty line

Reply:
Will take care of all below mentioned issues.